src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -2393,7 +2451,7 @@ private class CustomTodo : Todo
                 Assert.Equal(typeof(CustomTodo), parameter.ParameterType);
                 Assert.Equal("customTodo", parameter.Name);
 
-                var body = await context.Request.ReadFromJsonAsync<CustomTodo>();
+                var body = await JsonSerializer.DeserializeAsync<CustomTodo>(context.Request.Body);
```

Comment:
?

Reply:
`ReadFromJsonAsync` throws for non json content type, and for testing I am sending Json without a json content type. I can make a separate type for the test to make it more focused.